pypeit/spectrographs/gtc_osiris.py

```diff
@@ -276,58 +276,58 @@ def config_specific_par(self, scifile, inp_par=None):
 
         # Wavelength calibrations
         if self.get_meta_value(scifile, 'dispname') == 'R300B':
-            par['calibrations']['wavelengths']['wv_cen'] = 4405.
-            par['calibrations']['wavelengths']['disp'] = 4.96
+            # par['calibrations']['wavelengths']['wv_cen'] = 4405.
```

Comment:
@rcooke-ast -- you might confirm this is ok

Reply:
Yes, fine to comment it out, but maybe don't delete just yet - thanks!